src/client.rs

```diff
@@ -261,6 +268,17 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn check_result_mssing_found() -> Result<(), Box<dyn Error>> {
```

Comment:
Oh found a typo here!
```suggestion
    fn check_result_missing_found() -> Result<(), Box<dyn Error>> {
```